### DISCUSSION TITLE
[Solved] Vite config for iTwin packages

### DISCUSSION THREAD

**Discussion by LukePetko** [iTwin/itwinjs-core#4003] [Q&A] [answered]
Hi,

I'm trying to implement browser auth for an already existing project. I'm using Vite instead of React scripts in the project. As the package uses process to load environmental variables (not available in Vite), I get an error: "process is not defined."

Can anyone help me with what to add to the Vite config file?

Thanks

**Comment by aruniverse** [comment, marked as answer]
Have you seen this thread? 

https://github.com/iTwin/itwinjs-core/discussions/3429#discussioncomment-2456472
